Formula/cadence-workflow.rb

```diff
@@ -0,0 +1,33 @@
+class CadenceWorkflow < Formula
+  desc "Cadence Workflow is a distributed, scalable, durable, and highly available orchestration engine to execute asynchronous long-running business logic in a scalable and resilient way"
+  homepage "https://cadenceworkflow.io/"
+  url "https://github.com/uber/cadence.git",
```

Comment:
What is the relationship between this project and the one at https://github.com/onflow/cadence?

Reply:
There is no relationship at all. 
That is a programming language for blockchain stuff, while this is a service+programming framework for orchestration/workflow. 

Unfortunately, `cadence` is a very overloaded name. There is even one more `cadence` programming https://www.cadence.com/ which they don't make homebrew yet....

Reply:
We should probably still keep the `conflicts_with` line then.

Reply:
okay.

Reply:
Great, thanks.

When you're done making changes to this formula, and have squashed all the commits together, you'll need to push a separate commit that adds the appropriate `conflicts_with` line to the `cadence` formula as well.

Reply:
Added.